client/src/utils/api.js

```diff
@@ -0,0 +1,53 @@
+// File to contain methods to return data from API calls
+import axios from "axios";
+
+/**
+ * Get internships with given search options. Returns a promise that resolves to
+ * Internship[]
+ *
+ * @param {*} searchOptions - shape to be determined by backend
+ * @return {Promise<Internship[]>}
+ */
+export const getSearchInternships = (searchOptions) => {
```

Comment:
This file could get large. One solution would be to create a utils/api folder, with a barrel export in api.

/src/utils/api/index.js
export {
...
}

/src/utils/api/tracker
/src/utils/api/search or /src/utils/api/jobPostings, etc

Reply:
Def agree with this, left a note on team chat these are suggestions and as complexity grows, these could/should change